enterprise/frontend/src/embedding-sdk/cli/steps/ask-tenancy-columns.ts

```diff
@@ -0,0 +1,78 @@
+import { search } from "@inquirer/prompts";
+import toggle from "inquirer-toggle";
+
+import type { CliStepMethod } from "../types/cli";
+import { printHelperText } from "../utils/print";
+
+export const askForTenancyColumns: CliStepMethod = async state => {
+  printHelperText(
+    `e.g. does your table have a customer_id column to isolate tenants?`,
+  );
+
+  const hasTenancyIsolation = await toggle({
+    message: `Is your tenancy isolation based on a column?`,
+    default: true,
+  });
+
+  if (!hasTenancyIsolation) {
+    return [{ type: "success" }, state];
+  }
+
+  if (!state.chosenTables) {
+    const message = "You have not selected any tables.";
+
+    return [{ type: "error", message }, state];
+  }
+
+  const tenancyColumnNames: Record<string, string> = {};
+  let lastTenancyColumnName: string;
+
+  for (const table of state.chosenTables) {
+    const fieldChoices = [
+      // if the user's table of choice does not have any tenant id column, they can skip.
+      { name: "(no multi-tenancy column for this table)", value: null },
```

Comment:
Make this the top choice so it's easy to skip tables with no multi-tenancy columns.